07-alerting-and-notification/README.md

```diff
@@ -2,16 +2,21 @@
 
 ## Table of contents
```

Comment:
I would add a requirements or prerequisites section to this chapter as well. In it, you can refer to the Prometheus monitoring stack as the first requirement, then Emoji sample app, then Slack incoming webhook configuration, and any other requirement this chapter may have.

Don't forget to update TOC afterwards.